pkg/sysregistries/system_registries.go

```diff
@@ -0,0 +1,99 @@
+package sysregistries
+
+import (
+	"github.com/ghodss/yaml"
+	"io/ioutil"
+)
+
+/*
+systemRegistriesConfPath is the path to the system-wide registry configuration file
+and is used to add/subtract potential registries for obtaining images
+You can override this at build time with
+-ldflags '-X github.com/containers/image/sysregistries.systemRegistriesConfPath=$your_path'
+*/
+var systemRegistriesConfPath = builtinRegistriesConfPath
+
+/*
+builtinRegistriesConfPath is the path to registry configuration file
+DO NOT change this, instead see systemRegistriesConfPath above.
+*/
+const builtinRegistriesConfPath = "/etc/containers/registries.conf"
+
+var blank_string_array []string
+
+type registries struct {
+	Registries         []string `json:"registries"`
+	InsecureRegistries []string `json:"insecure_registries"`
+	BlockRegistries    []string `json:"block_registries"`
+}
+
+var blankFile = []byte("")
+
+/*
+Reads the global registry file from the filesystem. Returns
+a byte array
+*/
+func readRegistryConf() ([]byte, error) {
+	configBytes, err := ioutil.ReadFile(systemRegistriesConfPath)
+	if err != nil {
+		return blankFile, err
```

Comment:
seems like this is not fixed. it's still `return blankFile, err` - drop `blankFile` altogether also